src/amo/components/PermissionsCard/index.js

```diff
@@ -0,0 +1,280 @@
+              <Permission
+                className="hostPermission"
+                description={permissionStrings[moreKey](remaining)}
+                key={moreKey}
+              />
+            );
+          }
+        };
+
+        formatHostPermissions(wildcards, 'wildcard', 'tooManyWildcards');
+        formatHostPermissions(sites, 'oneSite', 'tooManySites');
+      }
+
+      // Next, show the native messaging permission if it is present.
+      const nativeMessagingPermission = 'nativeMessaging';
+      if (permissions.permissions.includes(nativeMessagingPermission)) {
+        permissionsToDisplay.push(
+          <Permission
+            className={nativeMessagingPermission}
+            description={permissionStrings[nativeMessagingPermission]}
+            key={nativeMessagingPermission}
+          />
+        );
+      }
+
+      // Finally, show remaining permissions, sorted alphabetically by the
+      // permission string to match Firefox.
+      const permissionsCopy = permissions.permissions.slice(0);
+      // eslint-disable-next-line no-restricted-syntax
+      for (const permission of permissionsCopy.sort()) {
+        // nativeMessaging is handled above.
+        if (permission === 'nativeMessaging') {
+          // eslint-disable-next-line no-continue
+          continue;
+        }
+        // Only output a permission if we have a string defined for it.
+        if (permissionStrings[permission]) {
+          permissionsToDisplay.push(
+            <Permission
+              className={permission}
+              description={permissionStrings[permission]}
+              key={permission}
+            />
+          );
+        }
+      }
+      return permissionsToDisplay;
+    }
+
+    // Get the permission strings from the appropriate file object.
+    const agentOsName =
+      userAgentInfo.os.name && userAgentInfo.os.name.toLowerCase();
+    const platform = userAgentOSToPlatform[agentOsName];
+    const file = addon.platformFiles[platform] || addon.platformFiles[OS_ALL];
+
+    if (!file) {
+      log.debug(oneLine`No file exists for platform "${agentOsName}"
+        (mapped to "${platform}"); platform files:`, addon.platformFiles);
+      return [];
+    }
+
+    const addonPermissions = file.permissions;
+    if (!addonPermissions.length) {
+      return null;
+    }
+
+    const content = formatPermissions(addonPermissions);
+    if (!content.length) {
+      return null;
+    }
+
+    return (
+      <Card
+        header={i18n.gettext('Permissions')}
+        className="PermissionsCard"
+      >
+        <div className="PermissionsCard-subhead">{i18n.gettext('This add-on can:')}</div>
+        <ul className="PermissionsCard-list">
+          {content}
+        </ul>
+        <Button
+          buttonType="neutral"
+          className="PermissionCard-learn-more"
+          href="https://support.mozilla.org/en-US/kb/permission-request-messages-firefox-extensions"
+          external
```

Comment:
I assume we want this to open in a new tab/window, but there doesn't seem to be anything currently in `Button` or `Link` to support that. I suppose I could use `customProps` in `Link`, but I expected there might be a property we could use to indicate we wanted to create a link that opens in a new tab/window.

Reply:
> I assume we want this to open in a new tab/window

I think it's fine to rely on our current implementation of buttons with the `external` property. I think we intentionally don't open it in a new window because that's something that users can easily decide to do on their own with command+click.